rero_ils/modules/update_logs/jsonschemas/update_logs/update_log-v0.0.1.json

```diff
@@ -0,0 +1,109 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "Update log",
+  "description": "Update log keeps the update history of a resource.",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid",
+    "fields"
+  ],
+  "propertiesOrder": [
+    "pid",
+    "record",
+    "creation",
+    "updates"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate update_log records against.",
+      "type": "string",
+      "minLength": 9,
+      "default": "https://ils.rero.ch/schema/update_logs/update_log-v0.0.1.json"
+    },
+    "pid": {
+      "title": "Update log PID",
+      "type": "string",
+      "minLength": 1
+    },
+    "record": {
+      "title": "Record",
+      "type": "object",
+      "properties": {
+        "$ref": {
+          "title": "record",
+          "type": "string",
+          "pattern": "^https://ils.rero.ch/api/acq_accounts|acq_invoices|acq_order_lines|acq_orders|budgets|circ_policies|collections|contributions|documents|holdings|ill_requets|item_types|items|libraries|loans|locations|notifications|organisations|patron_transaction_events|patron_transactions|patron_types|patrons|persons|templates|vendors/.*?$"
+        }
+      }
+    },
+    "creation": {
```

Comment:
Is `creation` field necessary ? Can we just keep the `updates` array field and consider than first update is the record creation (moreover  the structure of `updates` item seems the same than `creation`) ?